web/src/components/tasks/business-formation/review/ReviewBillingContact.tsx

```diff
@@ -10,24 +11,21 @@ export const ReviewBillingContact = (): ReactElement => {
   const italicNotEnteredText = `*${Config.formation.general.notEntered}*`;
 
   return (
-    <ReviewSection
-      buttonText={Config.formation.general.editButtonText}
-      header={Config.formation.sections.review.billingContactHeader}
-      stepName="Billing"
-      testId="edit-billing-contact-step"
-    >
+    <ReviewSubSection header={Config.formation.sections.review.billingContactHeader}>
       <ReviewLineItem
-        label={Config.formation.fields.contactFirstName.label}
+        label={`${Config.formation.fields.contactFirstName.label}:`}
         value={state.formationFormData.contactFirstName || italicNotEnteredText}
       />
       <ReviewLineItem
-        label={Config.formation.fields.contactLastName.label}
+        label={`${Config.formation.fields.contactLastName.label}:`}
```

Comment:
I'm seeing a number of different places where we're manually adding a `:` to the label. I don't think I'm seeing any where we don't. Should this just be part of the component?